tests/extensions/data/compare_to_zero.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=literal-comparison,missing-docstring,misplaced-comparison-constant
+# pylint: disable=literal-comparison,missing-docstring,misplaced-comparison-constant,invalid-name
```

Comment:
I find a little bit damageable if `pylint` is not compatible with constant name that are one letter long.
What do you think about it?